tests/Integration/ScriptsTest.php

```diff
@@ -402,4 +402,50 @@ public function assert_script_statuses( string $handle, array $assert_true = arr
 			);
 		}
 	}
+
+	/**
+	 * Test if the web stories analytics got enqueued.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @covers \Parsely\Scripts::run()
+	 * @group scripts
+	 */
+	public function test_web_stories_script_is_enqueued(): void {
+		self::$scripts->run();
```

Comment:
These tests should be moved to their own file - the code being called is not part of the Scripts class.

Reply:
Yes, the PR was a WIP :)